src/algebra/ring/ulift.lean

```diff
@@ -21,6 +21,16 @@ variables {α : Type u} {x y : ulift.{v} α}
 
 namespace ulift
 
+@[simp] lemma cast_nat_down [add_monoid_with_one α] :
```

Comment:
It looks like this is about `ulift.add_monoid_with_one`, which I think is defined in an earlier file; can they be adjacent?